data/transport missions.txt

```diff
@@ -3469,3 +3469,125 @@ mission "Inheritance Redirection"
+	on complete
+		dialog
+			`You wait for the Testament to land, but it remains in orbit.`
+			`There is no response to your hails. You need to take off and investigate.`
+
+mission "Hauler VI cargo prototype 2"
+	name "Escort Hauler prototype to a streetfood festival."
+	description "Escort the Testament to <destination> to deliver cooked fish dishes to a streetfood festival."
+	source "Skymoot"
+	destination "Zug"
+	landing
+	to offer
+		has "Hauler VI cargo prototype 1: done"	
+	npc accompany
+		government "Merchant"
+		personality escort timid
+		ship "Hauler III" "SB-Labs Testament"
+	on enter
+		dialog
+			`As soon as you take off, you receive an angry voice message over a short-range inter-ship communications channel: "Why were you trying to contact me? Does everyone need to know that we're here?"`
+			`She adds, "The cooling failed and the sushi nearly cooked. Let's go to <destination> to deliver it as fish dishes to a streetfood festival."`
+	on visit
+		dialog `You land on <destination>, but the Testament didn't catch up with you yet.`
+	on complete
+		dialog
+			`This time the Testament appears to be landing right behind you, but the ship flies away at full thrust just before touching the landing pad.`
+			`You receive a text message over the short-range inter-ship communication channel: "The sushi overcooked! We cannot be seen near the festival or near the shipyard with this overcooked sushi!"`
+			`The new destination is a floating factory on Mere.`
+
+mission "Hauler VI cargo prototype 3"
+	name "Escort Hauler prototype to factory"
+	description "Escort the Testament to <destination> to deliver overcooked fish to a floating catfood factory."
+	source "Zug"
+	destination "Mere"
+	landing
+	to offer
+		has "Hauler VI cargo prototype 2: done"
+	npc accompany
+		government "Merchant"
+		personality escort timid
+		ship "Hauler III" "SB-Labs Testament"
+	on visit
+		dialog `You land on <destination>, but the Testament didn't catch up with you yet.`
+	on complete
+		dialog
+			`The Testament enters the planets atmosphere at high speed and lands ahead of you near a catfood factory on one of the floating artificial islands.`
+			`Your infrared sensors show that the ship is emitting a lot of heat, but it's not coming from the thrusters, but rather from the cargo hold that appears to be seriously overheated.`
+			`Vanessa steps out of the ship and says, "The Sushi completely burned. We should unload it at the waste processing plant at Longjump."`
+			`She boards the Testament and takes off before you have time to respond. Apparently the cargo refridgeration systems are not working as well as Vanessa had hoped.`
+
+mission "Hauler VI cargo prototype 4"
+	name "Escort Hauler prototype to waste plant"
+	description "Escort the Testament to <destination> to deliver charred fish waste product to a waste-processing plant."
```

Comment:
`fish waste product` seems redundant... though just `fish waste` would be something entirely different

Reply:
changing to `charred food waste`